core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/blogPostAdminUtils.js

```diff
@@ -0,0 +1,371 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
Hmm, should this file be placed elsewhere, it seems to me that if we add file like this for every page this folder will be quite full.

Reply:
I'm not sure what exactly would it contain. Can you explain it more.

Reply:
for every user, there will be a utility file where all the functions of that user will be defined. and this is the utility file for that particular user. and I think we must place all those files inside this puppeteer-testing-utilities only.

Reply:
I think it makes sense to group users in one folder, yeah

Reply:
okay.. added a seperate folder `user-utilities`